src/convertToObject.js

```diff
@@ -12,7 +13,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString
+    .replace(/\s+/g, ' ')
+    .split(';')
+    .map(propertie => propertie.trim())
+    .filter(propertie => propertie)
```

Comment:
```suggestion
    .filter(Boolean)
```
result: `Boolean(property)` returns boolean value. You will learn about classes/constructors later, so it's just FYI